spec/models/job_spec.rb

```diff
@@ -13,6 +13,7 @@
 
       before(:each) { subject.valid? }
 
+      puts config.jobs_dir
```

Comment:
This line is causing the job specs to fail.  `config` is defined as a `let` block so there are some restrictions on when it can be called.  You can call it from inside an `it` block or inside a `before(:each)` block, but not directly inside a `context` block.